app/parse.py

```diff
@@ -15,8 +22,91 @@ class Product:
     num_of_reviews: int
 
 
+def get_soup(url):
+    content = requests.get(url).content
+    soup = BeautifulSoup(content, "html.parser")
+    return soup
+
+
+def get_data_from_soup(soup: BeautifulSoup, url: str):
+    if soup.select_one(".btn"):
+        driver = webdriver.Chrome()
+        driver.get(url)
+        actions = ActionChains(driver)
+        try:
+            button = driver.find_element(By.CSS_SELECTOR,
+                                         "a.btn.btn-lg.btn-block.btn-primary.ecomerce-items-scroll-more")
+            style = button.get_attribute("style")
+
+            while not style:
```

Comment:
The `while not style:` loop can potentially lead to an infinite loop if the style attribute is never set. Consider adding a condition to break out of the loop after a certain number of iterations or when a specific condition is met to prevent the program from hanging indefinitely.

Reply:
Please take into consideration this comment from ai